sklearn/linear_model/tests/test_base.py

```diff
@@ -362,7 +367,8 @@ def test_linear_regression_pd_sparse_dataframe_warning():
         reg.fit(df.iloc[:, 0:2], df.iloc[:, 3])
 
 
-def test_preprocess_data():
+def test_preprocess_data(global_random_seed):
+    rng = check_random_state(global_random_seed)
```

Comment:
updated for consistency: `check_random_state` is used in some functions, but not all

Reply:
```suggestion
    rng = np.random.RandomState(global_random_seed)
```